### PR TITLE
Project happy thoughts - Hang

### PR DESCRIPTION
**Pull request by Hang-Nguyen-Vu**
Link to my deployed project on Netlify: [hang-nguyen-happy-thoughts-project.netlify.app](url)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,22 @@
   </a>
 </h1>
 
-# Happy Thoughts
```
Comment (Cocofnas) [inline README.md:7]: I couldn't open the deployed link on netlify so can't test the functionality

File: src/App.jsx [hunk 2]
```diff
@@ -1,3 +1,59 @@
+import { useState, useEffect } from "react";
+import { Header } from "./Components/Header";
```
Comment (Cocofnas) [inline src/App.jsx:2]: Impressed that you only have four components!

File: src/App.jsx [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+      setMessageList([newMessage, ...messageList]);
+  }
+
+  return (
+    <div className="main-wrapper">
+      <Header />
+      <div className="message-components-wrapper">
+        <MessageForm addNewMessage={addNewMessage} fetchMessages={fetchMessages}/>
+        {loading ? (
+          <p className="loading">LOADING...</p>
```
Comment (Cocofnas) [inline src/App.jsx:52]: this is a nice feature

File: src/Components/MessageForm.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    const thoughtAPI = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts";
+
+    /* Function to handle change in text area */
+    const handleTextAreaChange = (e) => {
+        setNewMessage(e.target.value);
+    }
+
+    /* Function to check if the message is too long */
+    useEffect(() => {
+        if (newMessage.length >= 141) {
```
Comment (Cocofnas) [inline src/Components/MessageForm.jsx:18]: since I can't see, did you have a countdown under the insert textarea?

File: src/index.css [hunk 5]
```diff
@@ -11,3 +11,185 @@ code {
...
+.like-button {
+  border: none;
+  background-color: lightgrey;
+  border-radius: 50%;
+  cursor: pointer;
+  animation: pulse 2s infinite; /* Animation runs every 2 seconds an infinite amount of times */
+}
+
+/* Animation of the like button */
+@keyframes pulse {
```
Comment (Cocofnas) [inline src/index.css:145]: this is cute!

File: README.md (hunk 1 above)
Comment (Hang-Nguyen-Vu) [reply README.md:7]: It is wierd because I just tested it on my mobile and it worked fine :(

File: src/Components/MessageForm.jsx (hunk 4 above)
Comment (Hang-Nguyen-Vu) [reply src/Components/MessageForm.jsx:18]: I do, it works as per the instructions

**Pull request closed without merging**